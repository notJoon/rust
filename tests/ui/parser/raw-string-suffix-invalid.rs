// Test for issue #144161: Wrong error for string literal suffix when stuck together
// This tests the case where a raw string has an invalid suffix immediately followed by another string

fn main() {
    let s = r#" \\ "#r"\\ ";
    //~^ ERROR suffixes on string literals are invalid
    //~| ERROR expected one of `.`, `;`, `?`, `else`, or an operator
}
