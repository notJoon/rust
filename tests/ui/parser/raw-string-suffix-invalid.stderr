error: suffixes on string literals are invalid
  --> $DIR/raw-string-suffix-invalid.rs:5:13
   |
LL |     let s = r#" \ "#r"\ ";
   |             ^^^^^^^^^^ invalid suffix `r`

error: expected one of `.`, `;`, `?`, `else`, or an operator, found `"\ "`
  --> $DIR/raw-string-suffix-invalid.rs:5:23
   |
LL |     let s = r#" \ "#r"\ ";
   |                       ^^^^^ expected one of `.`, `;`, `?`, `else`, or an operator

error: aborting due to 2 previous errors

